lib/rules/color-hex-case/__tests__/index.js

```diff
@@ -164,3 +164,323 @@ testRule(
 		],
 	}),
 );
+
+testRule({
+	ruleName,
+	config: ['lower'],
+	syntax: 'css-in-js',
+	fix: true,
+	accept: [
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	stroke: "url(#gradiantA)",
+});
+`,
+			description: 'href with location',
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	color: "pink",
+});
+`,
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	color: "#000",
+});
+`,
+		},
+		{
+			code: `
```

Comment:
Let's combine the next three tests into valid CSS code, rather than use a `something` property, e.g.:

```css
background: linear-gradient(#aaa, #ffff, #0000ffcc);
```

This will cover the 4 and 8 digit tests.